Makefile

```
@@ -0,0 +1,81 @@
+#!/usr/bin/make -f
+
+.PHONY: env-start env-stop env-restart env-build env-destroy
+.PHONY: migrate upgrade seed lint
+.PHONY: test-users recreate-users upgrade-users migrate-usersrvice
+.PHONY: seed-users lint-users test-events recreate-events
+.PHONY: upgrade-events migrate-events lint-events
+
+PROJECT_NAME := 'my-dev-space'
+DOCKER_COMPOSE_FILE := ./docker-compose-dev.yml
+
+env-start:
```

`run` would be easier to reason about than `env-start`. 

e.g. `make run`

```suggestion
run:
```